### ISSUE TITLE
Replpace mootools and do not require bootstrap

### ISSUE THREAD

**Issue by Serhioromano** [Serhioromano/cobalt-9#144]
Cobalt will totally eluminate JQuery and bootstrap requirement to include it's own version

- [x] - jgrid.checkedout
- [x] - grid.sort

**Comment by KEZEO** [comment]
i think it' s good especially if that is help you to maintain cobalt developpement easy. maybe that could be great for helping 3dp developpers in template developpement too.

**Comment by Serhioromano** [comment]
task field in categories views either ajax order or sort.

**Comment by kiranetph** [comment]
how about an icon font?

**Comment by Serhioromano** [comment]
Yes, I am changing everything to icons. I create a custom font with fontello that contains only font items I need.

![2019-09-13_20-39-35](https://user-images.githubusercontent.com/650741/64871234-a1bc6f00-d666-11e9-9c11-3f76d0fbbbda.jpg)

![2019-09-13_20-40-18](https://user-images.githubusercontent.com/650741/64871270-b862c600-d666-11e9-831e-878f03d36a47.jpg)

A lot of work. I go through every file and there are a lot of them. One week over and I did not finish half. 

On the other hand, I like that I was able to review every line of code and refactor everything. I like the new look. 

I need suggestion about article control I'll create a post. Please check.
